app/styles/incarceration.tss

```diff
@@ -0,0 +1,64 @@
+".container": {
+	//backgroundColor:"#0E8FEA",
+	backgroundImage: "/global/background.png"
+}
+"#header": {
```

Comment:
I'm not totally sure we do, I think it could be extracted to a header file, but if we wanted to change the text in the header to reflect the page they were on we'd have to do that dynamically somewhere. So I guess it depends on what we decide.